### ISSUE TITLE
add ability to export worlds as Resonite package

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2288]
### Is your feature request related to a problem? Please describe.

Currently you cannot export worlds as a resonate package

### Describe the solution you'd like

I would like to export worlds as a resume package

### Describe alternatives you've considered

Exporting it as an object

### Additional Context

When I drag and drop the Resonite package of a world it should just open a new session with that world in it

### Requesters

@epicEaston197

**Comment by Frooxius** [comment]
What kind of usecases are you looking for this? 

We can add this, but we want to gauge how much interest there is in exporting entire worlds with this.

**Comment by epicEaston197** [comment]
> What kind of usecases are you looking for this?
> 
> We can add this, but we want to gauge how much interest there is in exporting entire worlds with this.

For example if you create a world you could sell it on sites like gum road

**Comment by ko-tengu** [comment]
Yeah, (generally paid) prefab worlds with quick customization options and preconfigured gimmicks is a healthy market for VRchat and I can absolutely see them being a popular choice for customized home worlds in Resonite also.

Arbitrary example: https://booth.pm/en/items/2579880

**Comment by Frooxius** [comment]
Is it something that people commonly do? Like publishing entire worlds like this.

The example seems like it's assets for building worlds with? Those could be exported as an item, unless I'm misunderstanding something.

**Comment by Nytra** [comment]
I can see a good use for this being if you want to distribute a world but you don't want to publish it or make it publicly accessible via URL. People would need the actual ResonitePackage file to access the world. It would be more secure.

**Comment by TisFoolish** [comment]
> Is it something that people commonly do? Like publishing entire worlds like this.

[Gumroad has a VRC tag for worlds, which includes premade worlds that people buy](https://gumroad.com/3d/vrchat/worlds?sort=hot_and_new)

**Comment by Frooxius** [comment]
Ah I see! Thanks for the links. We'll probably pick this up sooner or later.

I still recommend to give this issue a thumbs up, so we can kinda gauge the general interest in this.

**Comment by SlyTheFloof** [comment]
make it exportable so if you don't have the cloud space you can have the files on your pc

**Comment by epicEaston197** [comment]
This could also be really useful for world backups

**Comment by JackTheFoxOtter** [comment]
I think especially with Resonite's customizable nature, premade world "templates" for homes could probably be something reasonably popular. Also for the alternative usage of ResonitePackage files - local backups - being able to export entire worlds makes sense to me.
